imaging/ml/ml_codelab/breast_density_cloud_ml.ipynb

```diff
@@ -77,22 +77,83 @@
    "source": [
     "project_id = \"MY_PROJECT\" # @param\n",
     "location = \"us-central1\"\n",
+    "dataset_id = \"MY_DATASET\" # @param\n",
+    "dicom_store_id = \"MY_DICOM_STORE\" # @param\n",
+    "\n",
     "# Input data used by Cloud ML must be in a bucket with the following format.\n",
     "cloud_bucket_name = \"gs://\" + project_id + \"-vcm\""
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "UW3Y6Pd6d1Ey"
+   },
+   "outputs": [],
+   "source": [
+    "%%bash -s {project_id} {location} {cloud_bucket_name}\n",
+    "# Create bucket.\n",
+    "gsutil -q mb -c regional -l $2 $3\n",
+    "\n",
+    "# Allow Cloud Healthcare API to write to bucket.\n",
+    "PROJECT_NUMBER=`gcloud projects describe $1 | grep projectNumber | sed 's/[^0-9]//g'`\n",
+    "SERVICE_ACCOUNT=\"service-${PROJECT_NUMBER}@gcp-sa-healthcare.iam.gserviceaccount.com\"\n",
+    "COMPUTE_ENGINE_SERVICE_ACCOUNT=\"${PROJECT_NUMBER}-compute@developer.gserviceaccount.com\"\n",
+    "\n",
+    "gsutil -q iam ch serviceAccount:${SERVICE_ACCOUNT}:objectCreator $3\n",
+    "gsutil -q iam ch serviceAccount:${COMPUTE_ENGINE_SERVICE_ACCOUNT}:objectCreator $3\n",
+    "gcloud projects add-iam-policy-binding $1 --member=serviceAccount:${SERVICE_ACCOUNT} --role=roles/pubsub.publisher\n",
```

Comment:
these two cause a permission error. The error is because the service account does not have Project IAM Admin role. They would have to grant this in a different way. `does not have permission to access project [project_id:setIamPolicy] (or it may not exist): The caller does not have permission`

Reply:
everything else is good.

Reply:
in this case we'll need specify grant  Project IAM Admin permission to service account in readme, is it ok? 
or move all permission related commands to readme, but it may be too complicated for user since it includes just created buckets and datasets

Reply:
sounds good. Please add a link to the IAM page on console. and the IAM Admin permission that needs to be added!

Reply:
done